.circleci/config.yml

```diff
@@ -268,7 +268,19 @@ commands:
             cd server/scripts
             node deleteTwimlApp.js
 
-  ##
+  ###
+  # nvm commands
+  ###
+  set-nodejs-version:
+    steps:
+      - run:
+          name: "Set nodejs version to 18"
+          command: |
+            nvm install 18
```

Comment:
Do you want to do `18.19` to match line 92?

Reply:
I think it's OK to let nvm pull the latest 18. I only used `18.19` in the convenience image since that's what we use for the Test App, but it really shouldn't matter.